Converters/Rhino/Speckle.Converters.RhinoShared/ToSpeckle/Raw/CircleToSpeckleConverter.cs

```diff
@@ -30,10 +30,13 @@ IConversionContextStack<RhinoDoc, UnitSystem> contextStack
   /// ⚠️ This conversion assumes the domain of a circle is (0,1) as Rhino Circle types do not have a domain. If you want to preserve the domain use ArcCurve conversion instead.
   /// </remarks>
   public SOG.Circle Convert(RG.Circle target) =>
-    new(_planeConverter.Convert(target.Plane), target.Radius, _contextStack.Current.SpeckleUnits)
+    new()
     {
-      domain = new SOP.Interval(0, 1),
+      plane = _planeConverter.Convert(target.Plane),
+      radius = target.Radius,
+      units = _contextStack.Current.SpeckleUnits,
+      domain = SOP.Interval.UnitInterval,
       length = 2 * Math.PI * target.Radius,
-      area = Math.PI * Math.Pow(target.Radius, 2)
+      area = Math.PI * Math.Pow(target.Radius, 2),
```

Comment:
Shouldn't these trailing commas generate warnings? 🤔

Reply:
No, because its not a constructor call.
Not sure we have a preference.

Having a trailing  `,` makes for a cleaner diff when someone adds another property, but 🤷‍♂️ If we can't enforce it, then its whatever imo...

Reply:
Fair enough 🤐 I shall leave it as is then